src/SFA.DAS.FAT.Web/Models/EntryPoint.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace SFA.DAS.FAT.Web.Models
+{
+    public enum EntryPoint
+    {
+        Shortlist,
+        CourseDetail,
+        Providers,
+        ProviderDetail,

```

Comment:
Can these be explicitly numbered? See what Dan's done here for an example:
https://github.com/SkillsFundingAgency/das-employerdemand-api/pull/21/files#diff-807ca16d0141d67e129a32483671304cd490252d22e4fcaa335d68ad5b05a5e0